### PR TITLE
changes in tables names & validation

### PR DESCRIPTION
**Pull request by Karol2846**
I changed the table names and placement of validation annotations

### CODE REVIEW

File: src/main/java/com/example/JavaProject/entity/RecipeIngredient.java [hunk 1]
```diff
@@ -4,24 +4,20 @@
...
-import jakarta.validation.constraints.NotNull;
 
 import lombok.AllArgsConstructor;
 import lombok.Getter;
 import lombok.NoArgsConstructor;
 import lombok.Setter;
 
 @Entity
-@Table(name = "recipe_ingredient")
+@Table(name = "recipes_ingredients")
```
Comment (MajewskiOliwier) [inline src/main/java/com/example/JavaProject/entity/RecipeIngredient.java]: I prefer singular naming convention for database tables

Comment (MajewskiOliwier) [reply src/main/java/com/example/JavaProject/entity/RecipeIngredient.java]: Other than that everything is fine

Comment (Karol2846) [reply src/main/java/com/example/JavaProject/entity/RecipeIngredient.java]: Okey then, I changed them back to singular

Right now Is everything okey?

Comment (MajewskiOliwier) [reply src/main/java/com/example/JavaProject/entity/RecipeIngredient.java]: Yes, you can go ahead and merge it to master

**Review by MajewskiOliwier — APPROVED** [review]
Everything is fine now

**Pull request merged**